### DISCUSSION TITLE
ارور مهمparallel eror : persing of --jobs/-j/--max-procs/-p failed.

### DISCUSSION THREAD

**Discussion by armanem2** [MortezaBashsiz/CFScanner#88] [General]
سلام طریقه رفع این ارور هنگام اسکن ایپی بااین ارور عملا ایپی اسکن نمیشود مرسی

**Comment by MortezaBashsiz** [comment]
با چند پروسس همزمان اجرا میکنید؟

**Reply by kavehmashhori** [reply]
اموزش اسکن پایتون با ویندوز رو بزار لطفا
